packages/twenty-front/src/modules/context-store/hooks/useContextStoreObjectMetadataItemOrThrow.ts

```diff
@@ -10,11 +11,13 @@ export const useContextStoreObjectMetadataItemOrThrow = (
     contextStoreInstanceId,
   );
 
+  const isSettingsPage = useIsSettingsPage();
+
   const { objectMetadataItem } = useObjectMetadataItemById({
     objectId: objectMetadataItemId ?? '',
   });
 
-  if (!objectMetadataItem) {
+  if (!objectMetadataItem && !isSettingsPage) {
```

Comment:
no, this [useContextStoreObjectMetadataItemOrThrow.ts](https://github.com/twentyhq/twenty/pull/11206/files#diff-4496490d386214ac3a090677bf2791631aa9e9bcf4d26e82943132cb30dd73ae) should not be loaded if we are on a settings page. The higher the edge cases are treated in the hieararchy, the happier we are :)